web/carousel/src/components/ProductInSearch.js

```diff
@@ -0,0 +1,14 @@
+import styles from './ProductInSearch.module.css';
+
+const productInSearch = (props) => {
+    const product = props.product;
+    return (
+        <div className={styles.ProductInSearch}>
+            <img className={styles.Image} src={product.imageUrl} />
+            <p>{product.name}</p>
+            <p><b>{product.price}₺</b></p>
+        </div>
+    )
+}
```

Comment:
```
const productInSearch = ({ product }) => {
    const { imageUrl, name, price } = product ;
    return (
        <div className={styles.ProductInSearch}>
            <img className={styles.Image} src={imageUrl} />
            <p>{name}</p>
            <p><b>{price}₺</b></p>
        </div>
    )
}
```
I think with destructuring, it will be easier to read.